source/_docs/mu-plugin.md

```diff
@@ -0,0 +1,256 @@
+
+MU plugins are loaded by PHP, in alphabetical order, before normal plugins. This means API hooks added in an mu-plugin apply to all other plugins even if they run hooked-functions in the global namespace.
+
+## Why use MU Plugins?
+
+While you can add code in the `wp-config.php` file for site wide behavior, actions and filters shouldn't be added here.
+
+If they are added above the `require_once ABSPATH . 'wp-settings.php';` statement, the WordPress site will get a Fatal PHP error because the because `add_action()` and `add_filter()` functions won't be defined yet.
+
+If they are added below the `require_once ABSPATH . 'wp-settings.php';` statement, then the entirety of WordPress has already been loaded and the actions / filters won't be applied, or would be applied last.
+
+## Create Your MU Plugin
+1. Create a PHP file (i.e. `your-file.php`) in the `mu-plugins` folder (`code/wp-content/mu-plugins/your-file.php`).
+2. Provide the plugin details for its name, description etc.
+
+  ```php
+  <?php
+  /*
+    Plugin Name: Custom Actions and Filters
+    Plugin URI: https://plugin-site.example.com
+    Description: Boilerplate MU-Plugin for custom actions and filters to run for a site instead on setting in WP-config.php
+    Version: 0.1
+    Author: Pantheon
+    Author URI: https://yoursite.example.com
+  */
+  ```
+
+3. Add the custom functions, along with the filters or action that you want to run. Use the following script as a starting point for making your own plugin:
+
+  ```php
+  <?php
+  /*
+    Plugin Name: Custom Actions and Filters
+    Plugin URI: https://plugin-site.example.com
+    Description: Boilerplate MU-Plugin for custom actions and filters to run for a site instead on setting in WP-config.php
+    Version: 0.1
+    Author: Pantheon
+    Author URI: https://yoursite.example.com
+  */
+
+  if ( isset( $_ENV['PANTHEON_ENVIRONMENT'] ) ) {
+    // Actions or Filters that will only run in Pantheon .
+  };
+
+  if ( isset( $_ENV['PANTHEON_ENVIRONMENT'] ) && php_sapi_name() != 'cli' ) {
+  	// Add your actions or filters here that you want to exclude during WP CLI execution.
+  }
+
+  /*
+   * Set $regex_path_patterns accordingly.
+   *
+   * We don't set this variable for you, so you must define it
+   * yourself per your specific use case before the following conditional.
+   *
+   * For example, to exclude pages in the /news/ and /about/ path from cache, set:
+   *
+   *   $regex_path_patterns = array(
+   *     '#^/news/?#',
+   *     '#^/about/?#',
+   *   );
+   */
+  
+  // Loop through the patterns.
+  foreach ( $regex_path_patterns as $regex_path_pattern ) {
+    if ( preg_match( $regex_path_pattern, $_SERVER['REQUEST_URI'] ) ) {
+      add_action( 'send_headers', 'add_header_nocache', 15 );
+
+      // No need to continue the loop once there's a match.
+      break;
+    }
+  }
+
+  function add_header_nocache()
+  {
```

Comment:
Hey there! Tiny nitpick, but should this opening `{` be on it\'s own line?

Reply:
Unlined